common/baselines/tedlium2/data.py

```diff
@@ -0,0 +1,34 @@
+from collections import defaultdict
+from typing import Dict
+
+from i6_experiments.common.datasets.tedlium2.constants import CONCURRENT
+from i6_experiments.common.datasets.tedlium2.corpus import get_corpus_object_dict
+from i6_experiments.common.datasets.tedlium2.lexicon import (
+    get_g2p_augmented_bliss_lexicon,
+)
+from i6_experiments.common.setups.rasr.util import RasrDataInput
+
+from i6_experiments.users.luescher.setups.rasr.config.lex_config import (
+    LexiconRasrConfig,
```

Comment:
You are having a user import here in common.

Reply:
yes. that is in a different PR #110
only when #110 has been merged this will be changed

Reply:
This can be changed now right?